spec/models/greeter_role_spec.rb

```diff
@@ -10,33 +10,10 @@
 
 RSpec.describe GreeterRole, type: :model do
   describe "required fields" do
```

Comment:
I think there are now no fields at all, so consider deleting this `describe` block.